### PR TITLE
fix: use correct optimism icon in explore

### PR DESCRIPTION
**Pull request by cartcrom**
used preferred design confirmed by op team

### CODE REVIEW

File: src/components/Tokens/TokenTable/NetworkFilter.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ export default function NetworkFilter() {
       <NetworkFilterOption onClick={toggleMenu} aria-label={`networkFilter`} active={open}>
         <StyledMenuContent>
           <NetworkLabel>
-            <Logo src={circleLogoUrl ?? logoUrl} /> {label}
```
Comment (vm) [inline src/components/Tokens/TokenTable/NetworkFilter.tsx:109]: should we just kill the circle urls?

Comment (zzmp) [reply src/components/Tokens/TokenTable/NetworkFilter.tsx:109]: What else uses circle url?

File: src/constants/chainInfo.ts [hunk 2]
```diff
@@ -114,7 +113,8 @@ const CHAIN_INFO: ChainInfoMap = {
     infoLink: 'https://info.uniswap.org/#/optimism/',
     label: 'Optimism',
     logoUrl: optimismLogoUrl,
-    circleLogoUrl: optimismCircleLogoUrl,
+    // Optimism perfers same icon for both
+    circleLogoUrl: optimismLogoUrl,
```
Comment (zzmp) [inline src/constants/chainInfo.ts:117]: Considering that your updating the circleLogoUrl, I don't think you need to make anny changes to NetworkFilter.tsx

File: src/graphql/data/util.ts [hunk 3]
```diff
@@ -66,7 +66,7 @@ export const CHAIN_NAME_TO_CHAIN_ID: { [key: string]: SupportedChainId } = {
   OPTIMISM: SupportedChainId.OPTIMISM,
 }
 
-export const BACKEND_CHAIN_NAMES: Chain[] = ['ARBITRUM', 'CELO', 'ETHEREUM', 'OPTIMISM', 'POLYGON']
+export const BACKEND_CHAIN_NAMES: Chain[] = ['ETHEREUM', 'POLYGON', 'OPTIMISM', 'ARBITRUM', 'CELO']
```
Comment (zzmp) [inline src/graphql/data/util.ts:69]: back out this change - it looks unrelated.

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/graphql/data/util.ts (hunk 3 above)
Comment (cartcrom) [reply src/graphql/data/util.ts:69]: This does affect the order in which the ui displays them. I updated it so the explore network filter lists chains in the same order as the global network switcher. Since this doesn't affect the data components but makes UI better I think this is a relevant change. Ultimately a better solution could be to update chainInfo to include a backend chain name field instead of having this list, perhaps in sep PR

File: src/components/Tokens/TokenTable/NetworkFilter.tsx (hunk 1 above)
Comment (cartcrom) [reply src/components/Tokens/TokenTable/NetworkFilter.tsx:109]: logoUrl is for displaying chain switching options, like a main logo. CircleLogo provides circular image that is more visible laid over a token icon, used in the token table & details. LogoUrls don't look good overlaid on token icons because parts of them are transparent

File: src/constants/chainInfo.ts (hunk 2 above)
Comment (cartcrom) [reply src/constants/chainInfo.ts:117]: The filter needed to be updated to use logos instead of circleLogos, to match how the global network switcher displays

**Review by zzmp — APPROVED** [review]

File: src/graphql/data/util.ts (hunk 3 above)
Comment (awruble) [reply src/graphql/data/util.ts:69]: ah yes I had a sep ticket for this network ordering in the UI
